### PR TITLE
june dev update

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: content/posts/irods-development-update-june-2023.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Title: iRODS Development Update: June 2023
+Date: 2023-06-29 17:00
```
Comment (trel) [inline content/posts/irods-development-update-june-2023.md:2]: let's go with 6/30 10am

File: content/posts/irods-development-update-june-2023.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+Status: published
+
+
+Hello Everyone,
+
+The [iRODS User Group Meeting](https://irods.org/ugm2023) was a success!
+
+It was so wonderful being able to mingle with everyone and discuss various iRODS topics.
+
+This year we had more than 20 presentations (including lighting talks) and roughly 100 attendees spanning 10 countries, half of those being in-person.
```
Comment (trel) [inline content/posts/irods-development-update-june-2023.md:14]: "lighting" -> "lightning"

File: content/posts/irods-development-update-june-2023.md (hunk 1 above)
Comment (korydraughn) [reply content/posts/irods-development-update-june-2023.md]: done

File: content/posts/irods-development-update-june-2023.md (hunk 2 above)
Comment (korydraughn) [reply content/posts/irods-development-update-june-2023.md]: done

**Comment by korydraughn** [comment]
Rebased and squashed.

**Pull request merged**